fantasy_postseason_challenge/templates/select_team.html

```diff
@@ -8,7 +8,7 @@
     <body>
         <div class="main-wrapper">
             <h1>Select team for the Divisional Round!</h1><br>
-            <h4>Choose players from Rams @ Packers, Ravens @ Bills, Browns @ Chiefs, and Buccaneers @ Saints.</h4>
+            <h4>Choose players from Buccaneers @ Packers and Bills @ Chiefs.</h4>
             <h4>You must select one player from each team playing this weekend. Your roster will have two players from the same team.</h4><br>
```

Comment:
Change the instructions to: "You must select two players from each team playing this weekend. One team will have 3 players on your roster."